levels.go

```diff
@@ -1102,17 +1096,14 @@ func (s *levelsController) get(key []byte, maxVs *y.ValueStruct, startLevel int)
 		if vs.Value == nil && vs.Meta == 0 {
 			continue
 		}
-		if maxVs == nil || vs.Version == version {
+		if vs.Version == version {
```

Comment:
Earlier `maxVs == nil` was used to get the first found key? But the invariant that _entry with an older timestamp of the same key should not be written_ is no more valid.